packages/app/src/app/overmind/effects/vscode/ModelsHandler.ts

```diff
@@ -129,7 +129,9 @@ export class ModelsHandler {
   }
 
   public updateLineCommentIndication(model: any, lineNumber: number) {
-    const moduleModel = this.moduleModels[model.uri.path];
+    const moduleModel =
+      this.moduleModels[model.uri.path] ||
+      this.getModuleModelByPath(model.uri.path.replace('/sandbox', ''));
```

Comment:
Doesn't it make sense to completely remove `this.moduleModels[model.uri.path]` and enforce that we use `getModuleModelByPath`?